FABulous/FABulous.py

```diff
@@ -207,7 +209,7 @@ def common_options(
     )
 
     if (s := ctx.invoked_subcommand) and (
-        s.startswith("install") or s == "create-project" or s == "c"
+        s.startswith("install") or s.startswith("timing") or s == "create-project" or s == "c"
```

Comment:
It is better to allow context setup, so for example, the user can override the `sta` path or log the result to somewhere else.